GE_Requirements.mediawiki

```diff
@@ -2,24 +2,43 @@
+* Pull Requests MUST be public and open during a reasonable period of time so that the community can check them and contribute if necessary
 * Pull Requests need to be approved (using the standard Github function or a 'LGTM' comment) by all the reviewers involved, i.e. all people that gave their feedback on it.
-* Pull Request approval MUST be issued by a thrid party. Nobody can auto-merge their contributions.
+* Pull Request approval MUST be issued by a third party. Nobody can auto-merge their contributions.
 * Before merging an open Pull Request Travis status MUST be green. 
 
 
 == Publication Requirements ==
 
 * The SCM Tool MUST be [https://github.com Github.com]. 
-* Github MUST be used during the <b>whole development lifecycle</b>. See [[development.mediawiki#Development_Lifecycle|Development Lifecycle]] which describes a set of best practices for collaborative open source development.
-* See [[GE_Checklist.mediawiki|GE Publication Checklist]] for a list of tasks to be done in order to publish a new GE.
+* Github MUST be used during the <b>whole development lifecycle</b>. See [[development.mediawiki#Development_Lifecycle|Development Lifecycle Recommendations in FIWARE]]
+which describes a set of best practices for collaborative open source development.
+* See [[GE_Checklist.mediawiki|Checklist for publishing FIWARE Generic Enablers]] for a list of tasks to be done in order to publish a new GE.
+
+== Requirements regarding reference to FIWARE ==
+
+The README.md file of the GitHub repo, or several GitHub repos, associated to a FIWARE GE, '''MUST''' include an Introduction/About section at the beginning which includes:
+
+* the following paragraph clearly showing that the product has been labeled as a FIWARE GE and introducing FIWARE: 
+::'<name of the product> is a FIWARE Generic Enabler. Therefore, it  can be integrated as part of any platform “Powered by FIWARE”. FIWARE is a curated framework of open source platform components which can be assembled together with other third-party platform components to accelerate the development of Smart Solutions. You can find more info at the [https://deveopers.fiware.org FIWARE developers] website and the [https://fiware.org FIWARE] website.
+The complete list of FIWARE GEs and Incubated FIWARE GEs can be found at the [https://catalogue.fiware.org FIWARE Catalogue]'
+::
+* the [[project_badges.mediawiki|Project Badges]] which are stated as mandatory
+* what major.minor release of FIWARE each release of the product is compatible with.
+* The release note for each versioned release should include a list of the latest version of interoperable enablers it has been tested against as follows: 
+
+** "The FIWARE release X.Y.Z. of this product is fully compatible with all FIWARE GE components released as part of FIWARE release X.Y.Z. No interoperable breaking changes have been made since then."
```

Comment:
Suggested alternatives (I think that clearer for the reader):

"This release of {component} [note that this text appears in a release note so "this" identifies to which version I'm talking about] is part of FIWARE release {FIWARE_ver}. It if fully compatible with all other GE components in the same FIWARE release."

"This release of {component} [note that this text appears in a release note so "this" identifies to which version I'm talking about] is part of FIWARE release {FIWARE_ver}. It if fully compatible with all other GE components in the same FIWARE release, except in the following cases: {a bullet lit follows with the detail of incompatibilities}".

Regarding {FIWARE_ver} are the three numbers used to define the "compatibility set"? Sometimes I have heard that FIWARE 7 menas "compatibility based in NGSIv2" while FIWARE 8 means "compatibility based in NGSI-LD". Thus, I infer that only the first number (X) is used to define the compatibility sets and "Y" and "Z" aren't used.

Reply:
fixed 89696c2